temporalio/worker/_activity.py

```diff
@@ -532,6 +532,19 @@ async def _run_activity(
         except Exception:
             temporalio.activity.logger.exception("Failed completing activity task")
 
+    def assert_activity_valid(self, activity) -> Optional[str]:
```

Comment:
```suggestion
    def assert_activity_valid(self, activity) -> None:
```

And can change the first `return None` to just `return` and omit the last one

Reply:
Ah thanks, missed that change.